### ISSUE TITLE
Specifying inlineHeader in execution doesn't work

### ISSUE THREAD

**Issue by woj-tek** [mycila/license-maven-plugin#141]
Specifying `inlineHeader` within execution doesn't work:
```xml
<executions>
    <execution>
        <configuration>
            <inlineHeader>${project.name} - ${project.description}…
```
and yield
```
[INFO] --- license-maven-plugin:3.0:format (default-cli) @ tigase-acs ---
[INFO] Updating license headers...
[WARNING] No header file specified to check for license
```

**Comment by mathieucarbou** [comment]
Ho @woj-tek ,

Did you try to use a CDATA section ?

**Comment by woj-tek** [comment]
No, I wasn't. The idea was to define two executions (in parent pom) with different types of the licence headers and then activate appropriate one in child project.

And I worked around the issue:
I defined two properties with licence templates in parent and then reference one of those properties in child poms properties:
```xml
<properties>
	<license.inlineHeader>${inlineHeader_agpl3}</license.inlineHeader>
</properties>
```

IMHO in the end worked out better/simpler.
